exercises/practice/allergies/test_reporter.sql

```diff
@@ -1,6 +1,10 @@
 -- Update message for failed tests to give helpful information:
 UPDATE tests
-SET message = 'Result for ' || tests.task || ' ' || tests.item || ' ' || tests.score || ' is ' || actual.result || ', but should be ' || tests.expected
+SET message = (
+    'Result for ' || tests.task || ' ' || tests.item || ' ' || tests.score
+    || ' is <' || CASE WHEN actual.result IS NULL THEN "NULL" ELSE actual.result END
```

Comment:
```suggestion
    || ' is <' || COALESCE(actual.result, "NULL")
```

Reply:
This is the canonical way IIRC.

Reply:
Thank you for showing us the way! PR updated.